gremlin-dotnet/test/Gremlin.Net.IntegrationTest/Gremlin.Net.IntegrationTest.csproj

```diff
@@ -15,6 +15,7 @@
     <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="8.0.0" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.10.0" />
     <PackageReference Include="NSubstitute" Version="5.1.0" />
+    <PackageReference Include="System.Text.Json" Version="8.0.4" />
```

Comment:
Dependabot shouldn't just add new dependencies. This project should also already have this version as a peer dependency from the source project Gremlin.Net: https://github.com/apache/tinkerpop/blob/fd040c94a66516e473811fe29eaeaf4081cf104c/gremlin-dotnet/src/Gremlin.Net/Gremlin.Net.csproj#L77

Looks like a bug in Dependabot to me: dependabot/dependabot-core#10459